### DISCUSSION TITLE
Locking Layer makes all childs locked, even if one of them is unlocked.

### DISCUSSION THREAD

**Discussion by ar-at-localhost** [GrapesJS/grapesjs#5635] [Q&A]
I'm having problems with Layer Manager. Once I mark a layer as Locked, all of it's children are getting locked (by behavior, not by value) as well. What I was expecting to unlock a child layer and let the user edit it.

Is it by design? What's quickest way around it to make one or more child layers unlocked (by behavior, by value already unlocked)?

**Comment by artf** [comment]
Added this possibility [here](https://github.com/GrapesJS/grapesjs/commit/26dac0e69069ebe4fc87884ff07a2d9f808c3367)
